Formula/d/deno.rb

```diff
@@ -77,11 +77,7 @@ class Deno < Formula
   def install
     # Work around files missing from crate
     # TODO: Remove this at the same time as `rusty_v8` + `v8` resources
```

Comment:
Is the comment still valid after the cleanup?

Reply:
Yes. The cleanup is only related to Homebrew/brew#16951.